kartothek/io/dask/dataframe_cube.py

```diff
@@ -19,49 +37,94 @@
 )
 
 
+@default_docs
 def build_cube_from_dataframe(
-    data, cube, store, metadata=None, overwrite=False, partition_on=None
-):
+    data: Union[dd.DataFrame, Dict[str, dd.DataFrame]],
+    cube: Cube,
+    store: Callable[[], simplekv.KeyValueStore],
+    metadata: Optional[Dict[str, Dict[str, Any]]] = None,
+    overwrite: bool = False,
+    partition_on: Optional[Dict[str, Iterable[str]]] = None,
+    shuffle: bool = False,
+    num_buckets: int = 1,
+    bucket_by: Optional[Iterable[str]] = None,
+) -> Delayed:
     """
     Create dask computation graph that builds a cube with the data supplied from a dask dataframe.
 
     Parameters
     ----------
-    data: Union[dask.DataFrame, Dict[str, dask.DataFrame]
+    data:
```

Comment:
```suggestion
    data
```

Otherwise, sphinx will parse `Data that should be written...` (the next line) as a type, I think.